registry/conversion/conversion.go

```diff
@@ -33,6 +33,15 @@ func (cr *ConversionRegistry) RegisterFunctions(funcsMap sprout.FunctionMap) err
 	sprout.AddFunction(funcsMap, "toString", cr.ToString)
 	sprout.AddFunction(funcsMap, "toDate", cr.ToDate)
 	sprout.AddFunction(funcsMap, "toDuration", cr.ToDuration)
-	sprout.AddFunction(funcsMap, "mustToDate", cr.MustToDate)
+	return nil
+}
+
+func (cr *ConversionRegistry) RegisterAliases(aliasesMap sprout.FunctionAliasMap) error {
+	sprout.AddAlias(aliasesMap, "toDate", "mustToDate")
+	return nil
+}
+
+func (cr *ConversionRegistry) RegisterNotices(notices *[]sprout.FunctionNotice) error {
+	sprout.AddNotice(notices, sprout.NewDeprecatedNotice("mustToDate", "please use `toDate` instead"))
```

Comment:
Why not suggesting safeToDate ?

Or I'm missing something on what was mustToDate

Reply:
As discussed on #32, the default behavior will be "must" (stop rendering when an error occur instead of continue rendering without know an error has occurred in the template generation).

This is why we suggest `toDate` (having the same behavior of `mustToDate` by design). If users want to have a safe solution, users can activate the safe mode as described here : https://docs.atom.codes/sprout/~/changes/1eETtqCYtVeEPVLnbGs0/features/safe-functions

If this is ok for you I let you resolve conversions about this point 🙏